kura/org.eclipse.kura.linux.position/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,7 @@ Import-Package: javax.microedition.io,
  org.eclipse.kura.comm;version="[1.0,2.0)",
  org.eclipse.kura.configuration;version="[1.1,2.0)",
  org.eclipse.kura.net.modem;version="[1.0,2.0)",
- org.eclipse.kura.position;version="[1.1,1.2)",
+ org.eclipse.kura.position;version="[1.0,2.0)",
```

Comment:
Since linux.position is providing an implementation for the APIs, the range should be strict in the form [1.2, 1.3)